services/github/test/utils_test.js

```diff
@@ -318,6 +318,15 @@ suite(testing.suiteName(), function() {
     });
   });
 
+  suite('extractLogWithLongLine', function() {
+    test('should get the complete head lines corresponding to the max payload length', function() {
+      const payload = Array.from({ length: 10 }).map(line => `line: ${line}`);
+      payload.push('A'.repeat(100000))
+
+      assert.equal(extractLog(payload.join('\n'), 20, 200, 60000).length, 159)
```

Comment:
Thanks for adding this test! I wonder if a comment to explain all the numbers (array length 10, 20, 200, 60000, 100000 and 159) might be helpful.

Reply:
20/200 is what the github service uses, 60k is the max size it can use. 100k is just to overflow it, and 159 is because that's what makes the assert happy 🤣 (it's the 10 lines of logs I create)

I can add a comment sure